static/app/utils/replays/types.tsx

```diff
@@ -75,12 +75,20 @@ export function isErrorFrame(frame: ReplayFrame | undefined): frame is ErrorFram
   return Boolean(frame && 'category' in frame && frame.category === 'issue');
 }
 
-export function frameOpOrCategory(frame: ReplayFrame) {
+export function getFrameOpOrCategory(frame: ReplayFrame) {
   const val = ('op' in frame && frame.op) || ('category' in frame && frame.category);
   invariant(val, 'Frame has no category or op');
   return val;
 }
 
+export function isDeadClick(frame: SlowClickFrame) {
+  return (frame as SlowClickFrame).data.endReason === 'timeout';
+}
+
+export function isRageClick(frame: MultiClickFrame) {
+  return (frame as MultiClickFrame).data.clickCount >= 5;
```

Comment:
Should be isDeadClick && clickCount >= 5

Reply:
It seems like the cases are:
| Input crumb | properties | output title |
| --- | --- | --- |
| `ui.slowClick` | **time < 7s**, clicks < 5 | "Slow Click" | 
| `ui.slowClick` | **time >= 7s**, clicks < 5 | "Dead Click" |
| `ui.slowClick` | **time < 7s**, click >= 5 | ??? |
| `ui.slowClick` | **time >= 7s**, click >= 5 | "Rage Click" (aka: Dead, rageful, click) |
| `ui.multiClick` | **click < 5** | "Multi Click" (filtered from view) |
| `ui.multiClick` | **click >= 5** | "Rage Click" |

Reply:
| Breadcrumb category | properties | output title | visible in UI? |
| --- | --- | --- | --- |
| `ui.slowClick` | **time < 7s**<br/>clicks < 5 | "Slow Click" | No |
| `ui.slowClick` | **time >= 7s**<br/>clicks < 5 | "Dead Click" | No |
| `ui.slowClick` | **time < 7s**<br/>click >= 5 | ??? | No |
| `ui.slowClick` | **time >= 7s**<br/>click >= 5 | "Rage Click" (aka: Dead, rageful, click) |
| `ui.multiClick` | **click < 5** | "Multi Click" (filtered from view) | No |
| `ui.multiClick` | **click >= 5** | "Rage Click" | No |